uk.co.compucorp.civicrm.tasksassignments/js/test/filters/date-type.spec.js

```diff
@@ -20,33 +20,18 @@ define([
 
     describe('when filtering array of key dates with dateContactList property', function () {
       it('returns key dates that are of expected date types', function () {
-        expect(filter(withDateContactList, 'birth_date')).toEqual(filterDates(withDateContactList, 'birth_date'));
-        expect(filter(withDateContactList, 'period_end_date')).toEqual(filterDates(withDateContactList, 'period_end_date'));
-        expect(filter(withDateContactList, 'period_start_date')).toEqual(filterDates(withDateContactList, 'period_start_date'));
+        expect(filter(withDateContactList, 'birth_date').length).toEqual(10);
```

Comment:
1. Previously the data was being compared, why are we removing that? 
2. If we really need to remove the same, the test case description also needs to be renamed.

Reply:
> Previously the data was being compared, why are we removing that?

The `filterByDates` was removed because it was not giving the actual values we want. If we want to update the function to work as we expect the data, we would have to implement the code form `date-type.filter.js`. So, I removed it.